projects/packages/search/src/instant-search/components/tabbed-search-filters.scss

```diff
@@ -0,0 +1,25 @@
+@import '../lib/styles/_helper.scss';
+
+.jetpack-instant-search__search-static-filter-variation-tabbed {
+	border-bottom: 1px solid $color-layout-borders;
+	display: flex;
+
+	> div {
+		margin: 0;
+	}
+
+	.jetpack-instant-search__search-filter-list-input {
+		display: none;
+	}
+
+	.jetpack-instant-search__search-filter-list-label {
+		padding: 10px 20px;
+		margin: 0;
+	}
+
+	.jetpack-instant-search__search-filter-group-item {
+		&:has( :checked ) {
+			border-bottom: 3px solid var( --color-primary );
+		}
```

Comment:
Note that `:has` is currently [unsupported in Firefox](https://caniuse.com/css-has). This might be a good alternative:

```css
	.jetpack-instant-search__search-filter-list-input {
		&:checked + .jetpack-instant-search__search-filter-list-label {
			border-bottom: 3px solid var( --color-primary );
		}
	}
```